AK/QuickSort.h

```diff
@@ -88,27 +88,36 @@ void dual_pivot_quick_sort(Collection& col, int start, int end, LessThan less_th
 template<typename Iterator, typename LessThan>
 void quick_sort(Iterator start, Iterator end, LessThan less_than)
 {
-    int size = end - start;
-    if (size <= 1)
-        return;
-
-    int pivot_point = size / 2;
-    auto pivot = *(start + pivot_point);
-
-    if (pivot_point)
-        swap(*(start + pivot_point), *start);
+    for (;;) {
+        int size = end - start;
+        if (size <= 1)
+            return;
+
+        int pivot_point = size / 2;
+        auto pivot = *(start + pivot_point);
```

Comment:
Pls no copying the pivot?
I thought I made a PR with that change a while back 🤔

Reply:
That was already there, maybe you changed that in the dual_pivot_quick_sort?

Reply:
Ooooh, you're right!
I missed the single pivot one, it seems ^^'

Well...please make that a forwarding reference while you're at it :)

Reply:
Changed it, also included it in the test case.